TACC/Core-CMS
# Hotfix: Frontera: Correct path to recent import

# Overview

Update `taccsite_custom` after https://github.com/TACC/Core-CMS-Resources/pull/44.

# Testing

0. Set `.env` to `[…]=frontera-cms`.
1. Run `npm run build`.
2. Confirm __zero__ errors.
